bower.json

```diff
@@ -21,11 +21,11 @@
     "/tests/"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.0.0",
-    "dompurify": "^0.8.0"
+    "polymer": "~2.0.0",
+    "dompurify": "firmfirm/dompurify#v0.8.5"
```

Comment:
This fixates the version of dompurify to 0.8.5, right?
Why won't we allow minor and patch level updates?